src/components/tables/TableReducers.ts

```diff
@@ -122,12 +134,17 @@ export const tableReducer = (
         ...state,
         tableHeaderCellId: action.payload.id,
       };
+    case TableActions.updateSelectedIds:
+      selectedIds = action.payload.hasMultipleSelectedRow
+        ? _.union(state.data.selectedIds, action.payload.selectedIds)
+        : action.payload.selectedIds.slice(0, 1);
+      return { ...state, data: { ...state.data, selectedIds } };
     default:
       return state;
   }
 };
 
-export const tablesReducer = (tablesState = tablesInitialState, action: IReduxAction<IReduxActionsPayload>) => {
+export const tablesReducer = (tablesState = tablesInitialState, action: IReduxAction<ITableActionPayload | any>) => {
```

Comment:
you should leave IReduxActionsPayload, the tablesReducer receives all actions dispatched to the store, not only the ones related to the table

Reply:
if you want to make sure it extends to action that are outside react-vapor...i would consider adding `[key: string]: any` to IReduxActionsPayload instead of using `| any` here